api/envoy/extensions/common/dynamic_forward_proxy/v3/dns_cache.proto

```diff
@@ -100,5 +102,14 @@ message DnsCacheConfig {
   // Setting this value causes failure if the
   // ``envoy.restart_features.use_apple_api_for_dns_lookups`` runtime value is true during
   // server startup. Apple' API only uses UDP for DNS resolution.
-  bool use_tcp_for_dns_lookups = 8;
+  //
+  // .. attention::
```

Comment:
same comment here regarding this line (from previous dns_cache.proto instance)

Reply:
fixed